post_installation_configuration/vsphere-post-installation-encryption.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: ASSEMBLY
+[id="vsphere-post-installation-encryption"]
+= Enabling encryption on a vSphere cluster
+include::_attributes/common-attributes.adoc[]
+:context: vsphere-post-installation-encryption
+
+toc::[]
+
+You can encrypt your virtual machines after installing {product-title} {product-version} on vSphere by draining and shutting down your nodes one at a time. While each virtual machine is shutdown, you can enable encryption in the vCenter web interface.
+
+[id="encrypting-virtual-machines"]
+== Encrypting virtual machines
+
+You can encrypt your virtual machines with the following process. You can drain your virtual machines, power them down and encrypt them using the vCenter interface. Finally, you can create a storage class to use the encrypted storage.
+
+.Prerequisites
+
+* You have configured a Standard key provider in vSphere. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-AC06B3C3-901F-402E-B25F-1EE7809D1264.html[Adding a KMS to vCenter Server].
++
+[IMPORTANT]
+====
+The Native key provider in vCenter is not supported. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-54B9FBA2-FDB1-400B-A6AE-81BF3AC9DF97.html[vSphere Native Key Provider Overview].
+====
+
+* You have enabled host encryption mode on all of the ESXi hosts that are hosting the cluster. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-A9E1F016-51B3-472F-B8DE-803F6BDB70BC.html[Enabling host encryption mode].
+* You have a vSphere account which has all cryptographic privileges enabled. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-660CCB35-847F-46B3-81CA-10DDDB9D7AA9.html[Cryptographic Operations Privileges].
+
+.Procedure
+
+. Drain and cordon one of your nodes. For detailed instructions on node management, see xref:../nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-evacuating_nodes-nodes-working[Working with nodes].
+. Shutdown the virtual machine associated with that node in the vCenter interface.
+. Right-click on the virtual machine in the vCenter interface and select *VM Policies* -> *Edit VM Storage Policies*.
+. Select the encryption storage policy named "VM Encryption Policy" and select *OK*.
```

Comment:
We should avoid specifying "VM Encryption Policy" specifically, because in vSAN environment the policy's name is different. We should rather say - "Select a suitable storage policy that support VM encryption on selected datastore".